### DISCUSSION TITLE
Can tauri2 create a picture tray menu function using only the JavaScript API: @tauri-apps/api/tray library?

### DISCUSSION THREAD

**Discussion by muyaCode** [tauri-apps/tauri#10396] [Q&A]
Can tauri2 create a picture tray menu function using only the JavaScript API: @tauri-apps/api/tray library? 
How to listen to the tray menu click?
I can't find it in the documentation：[tauri2-namespacetray](https://v2.tauri.app/reference/javascript/api/namespacetray/)

**Comment by FabianLars** [comment]
I'm sorry but could you explain a bit more what you mean? Maybe a screenshot of another app that does what you want?

**Comment by charlesschaefer** [comment]
Maybe this can help you? https://v2.tauri.app/reference/javascript/api/namespacemenu/
